cli/cmd/check.go

```diff
@@ -149,6 +152,8 @@ func configureAndRunChecks(wout io.Writer, werr io.Writer, stage string, options
 		checks = append(checks, healthcheck.LinkerdPreInstallChecks)
 		if !options.cniEnabled {
```

Comment:
Consider flipping the if and else blocks here to avoid the negation.

Reply:
Wait, actually, don't we always want to run the LinkerdPreInstallCapabilityChecks, even when CNI is enabled?  This check will ensure that the Linkerd components (excluding the CNI ones) don't already exist.

Reply:
I dont think so, the `LinkerdPreInstallCapabilityChecks` checks check for NET_ADMIN and NET_RAW. We do not need these if we have CNI runnig. Correct ?

Reply:
ah, I was getthing this mixed up with the check to ensure cluster scoped resources like ClusterRoles did not already exist.  nevermind